src/modules/universal-discovery/components/selected-content/selected.content.component.js

```diff
@@ -26,7 +26,7 @@ export default class SelectedContentComponent extends PureComponent {
      * @memberof SelectedContentComponent
      */
     togglePopup() {
-        this.setState((state) => ({ isPopupVisible: !state.isPopupVisible && !!state.items.length }));
+        this.setState((state) => ({ isPopupVisible: !state.isPopupVisible && !!this.props.items.length }));
```

Comment:
I'm not sure if it matters here, but maybe:
```js
this.setState((state, props) => ({ isPopupVisible: !state.isPopupVisible && !!props.items.length }));
```

Reply:
Nice catch!

Reply:
In this case, it doesn't but fixed.